docs/docs/how-tos/How to authenticate using Keycloak.md

```diff
@@ -0,0 +1,86 @@
+---
+id: login-keycloak
+title: How to authenticate using Keycloak and launch your server
+description: |
+  This is a how-to for authenticating using Keycloak.
+---
+
+# How to authenticate using Keycloak and launch your server
+
+This guide aims to give a basic overview of the Nebari login process. Your organization's Nebari deployment will likely have a slightly different process due to the many
+authentication providers available for Nebari can within Keycloak.
+
+## 1. Connect to your Nebari cluster
+
+For example, your cluster domain should look like `https://nebari-demo.nebari.dev`.
+
+Once on the site, you will be prompted by a login, similar to
+the login page shown in the image below.
+
+![Nebari login screen](/img/how-tos/nebari_login_screen.png)
+
+Nebari now uses an open source tool called [Keycloak](https://www.keycloak.org/) for user management. This makes it a little challenging to detail the exact process as it might differ greatly between
+authentication providers ([LDAP](https://pt.wikipedia.org/wiki/LDAP), [OAuth 2.0](https://oauth.net/2/), passwordless authentication, password-based authentication and many others). For more information on how to configure Keycloak, and add new users, make sure to check [How to configure Keycloak](/how-tos/configuring-keycloak) sections of our docs.
+
+:::warning
+As Nebari **does not** create a Keycloak user for you during deployment, the following steps assumes that your Nebari administrator have already created a Keycloak user in the admin console for you. If not, follow the steps in [Adding new users to Keycloak](/how-tos/configuring-keycloak#adding-a-nebari-user) to create a Keycloak user.
+:::
```

Comment:
Maybe this should be moved higher up in the document under a "Pre-requisites" section